.travis.yml

```diff
@@ -71,6 +72,9 @@ script:
      make check V=1 BINARY=pcvc4 CVC4_REGRESSION_ARGS='--fallback-sequential --no-early-exit' RUN_REGRESSION_ARGS= ||
        error "PORTFOLIO TEST FAILED";
    }
+   JavaApiTest() {
+     cd test/java && ant && cd ../..
```

Comment:
Let's do at least the commands independently. If ant fails, we should still go back to ../..